EdFi.Ods.Utilities.Migration/Scripts/MsSql/02Upgrade/v52_to_v53_Changes/03 DeployJournal/v53FeaturesUpdateDeployJournal.sql

```diff
@@ -0,0 +1,31 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE @EdFiDeployJournal TABLE
+(
+[ScriptName] [nvarchar] (255) NOT NULL
+)
+
+INSERT @EdFiDeployJournal (ScriptName)
```

Comment:
SqlServerV52ToV53JournalTests.cs needs to enable ChangeQueries feature as like below ,so that it will  execute also change query feature in migration scripts and validate the journal entries including change query feature from 5.3 database reference 

protected override List<EdFiOdsFeature> FeaturesBeforeUpgrade =>  new List<EdFiOdsFeature> { EdFiOdsFeature.ChangeQueries };